syno-plex-update.sh

```diff
@@ -89,6 +93,14 @@ function download_release_metadata {
     # Grab release information from Plex API (as JSON).
     local release_url=${PLEX_RELEASE_API/TokenPlaceholder/$(get_plex_token)}
     local response
+    
+    is_public_channel=$(is_public_channel)
+    if [ "$is_public_channel" -eq "0" ]; then
+        write_log 'Public channel'
```

Comment:
My preferences file (free tier) doesn't have this substring and this line raises an error:
```
line 98: [: : integer expression expected
```

I think the function needs something like a `grep -oP ... || echo "0"` to handle the unhappy case.